packages/core/src/components/graphs/line.ts

```diff
@@ -145,7 +146,7 @@ export class Line extends Component {
 
 	handleLegendMouseOut = (event: CustomEvent) => {
 		this.parent
-			.selectAll("path.line")
+			.selectAll(`g#coverClip`)
```

Comment:
what does this do?

Reply:
Good catch! These should be deleted.
Thanks.